### PR TITLE
List Numbering Fix

### PR DESCRIPTION
**Pull request by YuriySamorodov**
1. List Numbering Fix
2. Added some clarification to one of the notes section

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/tmr-meeting-recording-change.md [hunk 1]
```diff
@@ -56,29 +56,29 @@ Watch "Meeting Recording" for more information.
 The meeting recording option is a setting at the Teams policy level. The following example shows how to set the Global policy. Make sure that you set the meeting recording option for the policy or policies that you've assigned to your users.
 
 > [!Note]
-> Teams meeting policy changes take awhile to propagate. Check back after a few hours of setting it, then sign out and sign in again.
+> Teams meeting policy changes take awhile to propagate. Check back after a few hours of setting it, then sign out and sign in Teams Desktop app again or simply restart computer.
```
Comment (JohanFreelancer9) [inline Teams/tmr-meeting-recording-change.md:59]:
```suggestion
> Teams meeting policy changes take a while to propagate. Check back after a few hours of setting it, then sign out and sign in to the Teams Desktop app again or simply restart your computer.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Teams/tmr-meeting-recording-change.md (hunk 1 above)
Comment (YuriySamorodov) [reply Teams/tmr-meeting-recording-change.md]: @JohanFreelancer9 , this is completed. Thanks a bunch!

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**